### PR TITLE
Add support for Icotera

### PR DESCRIPTION
**Pull request by funzoneq**
## Pre-Request Checklist

- [X] Passes rubocop code analysis (try `rubocop --auto-correct`)
- [ ] Tests added or adapted (try `rake test`)
- [X] Changes are reflected in the documentation
- [X] User-visible changes appended to [CHANGELOG.md](/CHANGELOG.md)

## Description

Adding support for [Icotera](https://icotera.com/products) devices.

### CODE REVIEW

**Comment by wk** [comment]
Looks good! Please update the `docs/Supported-OS-Types.md` file as part of this PR with a vendor and model (and link to this model file) so that users can discover this device is supported!

**Comment by funzoneq** [comment]
@wk I updated the docs :)

**Comment by wk** [comment]
Thanks! I'll add a more detailed review with a few remarks in a moment, and once those are addressed, this can go in!

File: lib/oxidized/model/icotera.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    cfg.gsub! /^\s+Bcast in.*\n/, ""
+    cfg.gsub! /^\s+Mcast in.*\n/, ""
+    cfg.gsub! /^\s+Ucast in pps.*\n/, ""
+    cfg.gsub! /^\s+Mcast in pps.*\n/, ""
+    cfg.gsub! /^\s+Total in bps.*\n/, ""
+
+    comment cfg
+  end
+
+  cmd 'copy progress screen' do |cfg|
```
Comment (wk) [inline lib/oxidized/model/icotera.rb]: This block is equivalent to simply doing `cmd 'copy progress screen'`, as no processing happens in the block.

File: lib/oxidized/model/icotera.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    cfg
+  end
+
+  cfg :telnet, :ssh do
```
Comment (wk) [inline lib/oxidized/model/icotera.rb]: Here, `:telnet` is defined, but there's no telnet username or password prompt definitions in the model (those are different from SSH, as SSH is typically non-interactive and username/password are fed pragmatically. You might want to remove `:telnet`, or identify and add the prompts.

**Comment by funzoneq** [comment]
@wk I addressed your comments. Please review.

**Comment by wk** [comment]
Thanks!

**Pull request merged**